### ISSUE TITLE
Haste and Flurry II should not be stackable.

### ISSUE THREAD

**Issue by Aeltorian** [CatsAndBoats/catseyexi#295]
[Bug Report](https://discord.com/channels/696847769444548700/1168698854397444166)

It's reported that [Haste](https://www.bg-wiki.com/ffxi/Haste) and [Flurry II](https://www.bg-wiki.com/ffxi/Flurry_II) are stacking on each other. This isn't retail behavior. [Flurry II](https://www.bg-wiki.com/ffxi/Flurry_II) should be overridden by [Haste](https://www.bg-wiki.com/ffxi/Haste)

**Comment by Thorinori** [comment]
This is incorrect, Flurry II is overriden by Haste II in retail, but not Haste I. BG-Wiki is wrong in this case. Flurry I and Haste I do overwrite each other on retail though, neither has priority over the other, just most recent cast negates the other.
Source: Personally tested on retail

![image](https://github.com/CatsAndBoats/catseyexi/assets/8883561/0b8c916f-142c-4bd0-810a-3a3601b4dbb8)

(Timers addon is incorrect, Flurry 2 icon still present at the top after casting Haste)

**Comment by Thorinori** [comment]
The stacking part is a problem still though, the overriding is what is incorrect.
